Experimental/DedicatedGameServer/Assets/Editor/Tutorials/Tutorial1_SetupMultiplayerPlayMode/2-Create a Virtual Player.asset

```diff
@@ -0,0 +1,197 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ff771ccdf4150419d9ff4d342b069aae, type: 3}
+  m_Name: 2-Create a Virtual Player
+  m_EditorClassIdentifier: 
+  Title:
+    m_Untranslated: Multiplayer Play Mode Window
+  m_Paragraphs:
+    m_Items:
+    - m_Type: 6
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: 
+      m_CodeSample: 
+      m_PostInstructionImage: {fileID: 0}
+      m_Tutorial: {fileID: 0}
+      m_Image: {fileID: 0}
+      m_VideoUrl: 
+      m_Video: {fileID: 0}
+      m_CriteriaCompletion: 0
+      m_Criteria:
+        m_Items: []
+      m_MaskingSettings:
+        m_MaskingEnabled: 1
+        m_UnmaskedViews:
+        - m_SelectorType: 1
+          m_ViewType:
+            m_TypeName: UnityEditor.Toolbar, UnityEditor.CoreModule, Version=0.0.0.0,
+              Culture=neutral, PublicKeyToken=null
+          m_FocusEditorWindow: 1
+          m_EditorWindowType:
+            m_TypeName: Unity.Multiplayer.Playmode.WorkflowUI.Editor.MultiplayerWindow,
+              Unity.Multiplayer.Playmode.WorkflowUI.Editor, Version=0.0.0.0, Culture=neutral,
+              PublicKeyToken=null
+          m_AlternateEditorWindowTypes:
+            m_Items: []
+          m_MaskType: 0
+          m_MaskSizeModifier: 0
+          m_UnmaskedControls: []
+      m_Summary: 
+      m_Description: 
+      m_InstructionBoxTitle: 
+      m_InstructionText: 
+      m_TutorialButtonText: 
+    - m_Type: 0
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: "To open the Multiplayer Play Mode window:  \n\n1. Go to
+          <b>Window</b> > <b>Multiplayer</b> > <b>Multiplayer Play Mode</b>.\n\n2.
+          Select the checkbox for Virtual Player 2. This player opens when you enter
+          Play mode.\n\n3. Wait for the Player status to change from inactive to
+          active for Virtual Player 2.\n\nWhen active, the Virtual Player(s) appear
+          in Play Mode in the main Editor Player window. A separate window opens
+          for each enabled Player with the Player name as the window title."
```

Comment:
This page has 3 instructions in it ("open the window > enable a virtual player > wait and then press next"), which makes handholding difficult. In-Editor Tutorials are meant to be "step-by-step" and **to validate** that the user has done what you're asking them to do. I'd break down the "open MPPM" and "enable virtual player + wait" in two different pages.

Reply:
Split this into two tutorial pages!